dnf-behave-tests/dnf/broken-dependencies-report.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Reporting broken dependencies with various strict, best options
+
+
+@bz2088422
+Scenario: Broken dependencies block the transaction when strict mode is on
+  Given I use repository "dnf-ci-fedora"
+   When I execute dnf with args "install postgresql --exclude=postgresql-libs --setopt=strict=true"
+   Then the exit code is 1
+
+
+@bz2088422
+Scenario: Broken dependencies are reported when strict and best options are off
+  Given I use repository "dnf-ci-fedora"
+   When I execute dnf with args "install postgresql --exclude=postgresql-libs --setopt=strict=false --setopt=best=false"
+   Then the exit code is 0
+    And stderr contains "package postgresql-9.6.5-1.fc29.x86_64 requires postgresql-libs"
```

Comment:
Could you check the full `stderr` output with `stderr is` step?
We don't want to miss any potentially important outputs.